client/server/routes/search/rfc-search-utils.ts

```diff
@@ -71,11 +71,11 @@ export const translateParamsObject = (
       searchParamsObj.to = `${legacySearchObj.to_year}-${monthNameToNumber(legacySearchObj.to_month, 1)}`
     }
   } else if (legacySearchObj.pub_date_type === 'this_month') {
-    const now = DateTime.utc()
+    const now = DateTime.now()
```

Comment:
Does the distinction between utc and system time make a difference here? Hoping we're not getting different results depending on the browser's time zone

Reply:
This is server code so server timezone only. The bug was using both utc and local, and the fix was to use only one so I chose local because the the vitest `vi.setSystemTime` in `legacy-search-redirect.test.ts` expects a Date which is local not utc.